extensions/gadgets/TestGadget/protractor.js

```diff
@@ -0,0 +1,39 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview End-to-end testing utilities for the TestGadget gadget.
+ */
+
+// TODO(anuzis): Oppia's current skin doesn't have a gadget panel with
```

Comment:
I'm assuming the Test Gadget exists for testing purposes and wouldn't be user facing. If so would there ever be a need for it to have its own integration tests?